pkg/webhooks/handlers/admission.go

```diff
@@ -77,24 +76,16 @@ func Admission(logger logr.Logger, inner AdmissionHandler) http.HandlerFunc {
 }
 
 func Filter(c config.Configuration, inner AdmissionHandler) AdmissionHandler {
-	return func(request *admissionv1.AdmissionRequest) *admissionv1.AdmissionResponse {
+	return func(logger logr.Logger, request *admissionv1.AdmissionRequest) *admissionv1.AdmissionResponse {
 		if c.ToFilter(request.Kind.Kind, request.Namespace, request.Name) {
 			return nil
 		}
-		return inner(request)
+		return inner(logger, request)
 	}
 }
 
 func Verify(m *webhookconfig.Monitor, logger logr.Logger) AdmissionHandler {
-	return func(request *admissionv1.AdmissionRequest) *admissionv1.AdmissionResponse {
-		logger = logger.WithValues(
-			"action", "verify",
-			"kind", request.Kind,
-			"namespace", request.Namespace,
-			"name", request.Name,
-			"operation", request.Operation,
-			"gvk", request.Kind.String(),
-		)
```

Comment:
Same here.

Reply:
This method has been deleted, the logger is now created in the `Admission` handler util.